### DISCUSSION TITLE
Templates para o blogtext

### DISCUSSION THREAD

**Discussion by EdilsonBaggio** [rafaelleitedasilva/blogtext#10] [Ideas]
- [ ] https://demo.templatesjungle.com/techlight/single-post.html
- [x] https://themewagon.github.io/blogy/single.html

**Comment by rafaelleitedasilva** [comment]
https://themewagon.github.io/blogy/index.html

Acho que a gente vai conseguir ter uma liberdade maior utilizando esse template.

**Comment by andreparelho** [comment]
Curti os dois mas como tenho pouca noção deixo com vocês kkkkkkkk o primeiro template tem cara de mais minimalista

**Comment by rafaelleitedasilva** [comment]
Bora usar o segundo como premissa e talvez mesclar o outro em uma ou duas páginas se necessário.
